src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -27,8 +28,9 @@ public void execute() {
     double xSpeed = m_driver.getForwardTranslation();
     double ySpeed = m_driver.getSideTranslation();
     double rot = m_driver.getRotation();
+    double slowFactor = m_driver.isSlowSpeed().getAsBoolean() ? 0.5 : 1;
 
-    m_drive.drive(xSpeed, ySpeed, rot, true);
+    m_drive.drive(xSpeed * slowFactor, ySpeed * slowFactor, rot, true);//m_driver.isSlowSpeed().getAsBoolean()
```

Comment:
All this slow speed stuff should be in a separate PR so it can be approved quickly